clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -164,8 +164,8 @@ public Struct parseResponse(short version, ByteBuffer buffer) {
             DescribeConfigsResponse.schemaVersions()),
     ALTER_CONFIGS(33, "AlterConfigs", AlterConfigsRequest.schemaVersions(),
             AlterConfigsResponse.schemaVersions()),
-    ALTER_REPLICA_DIR(34, "AlterReplicaDir", AlterReplicaDirRequest.schemaVersions(),
-            AlterReplicaDirResponse.schemaVersions()),
+    ALTER_REPLICA_LOG_DIRS(34, "AlterReplicaDirs", AlterReplicaLogDirsRequest.schemaVersions(),
```

Comment:
AlterReplicaDirs => AlterReplicaLogDirs ?

Reply:
Sorry, I should have noticed this. Thanks so much for your detailed review!

Reply:
I have updated the patch to fix it.